index.js

```diff
@@ -156,8 +156,18 @@ Plugin.prototype.readFile = function(file, callback) {
 			middleware.fileSystem.readFile("/_karma_webpack_/" + file.replace(/\\/g, "/"), callback);
 		}
 	}
-	if(!this.waiting)
-		doRead();
+	if (!this.waiting) {
```

Comment:
I wonder why this fix is needed. Shouldn't be `this.waiting` truthy until all files are available?